conf/dynaconf_hooks.py

```diff
@@ -127,7 +127,7 @@ def get_dogfood_satclient_repos(settings):
 
 def get_ohsnap_repo_url(settings, repo, product=None, release=None, os_release=None, snap=''):
     repourl = dogfood_repository(
-        settings.repos.ohsnap_repo_host,
+        settings.ohsnap,
```

Comment:
```suggestion
       settings.ohsnap.host,
```

Reply:
@Gauravtalreja1 , no, this one is correct. The method now accepts the `ohsnap` dictionary as it uses the rest of the settings further.
Same applies to your other suggestions